partition_manager.py: Change span size calculation to include alignment (#11449)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

This section provides detailed lists of changes by :ref:`script <scripts>`.

* :ref:`partition_manager`:

>   * Change the size of span partitions to include alignment paritions (EMPTY_x) so that the size reflects the size taken in memory, not just the sum of sizes of the included partitions.

Reviewer: ```suggestion
  * The size of the span partitions was changed to include the alignment paritions (``EMPTY_x``).
  The size of the span partitions now reflects the memory space taken, not just the sum of the sizes of the included partitions.
```

Author: Done, I also added a bit more clarifying information
